src/huggingface_hub/utils/common.py

```diff
@@ -0,0 +1,37 @@
+"""Miscellaneous / general utilities"""
```

Comment:
utils/common sounds like this file will end up with bunch of things. Maybe I would name this file `_chunk_utils`? Not sure

Reply:
Addressed in 49ca725